kubernetes/.swagger-codegen-ignore

```diff
@@ -1,3 +1,6 @@
 .gitignore
 git_push.sh
+# Remove this once https://github.com/swagger-api/swagger-codegen/pull/5629
```

Comment:
It needs to be released so we remove this. I think the comment should say "until 2.2.3 is released or something like that".